File: app/src/main/java/nl/rsdt/japp/jotial/maps/wrapper/osm/OsmCameraPosition.java
package nl.rsdt.japp.jotial.maps.wrapper.osm;

import org.osmdroid.views.MapView;

import nl.rsdt.japp.jotial.maps.wrapper.ICameraPosition;

/**
 * Created by mattijn on 17/08/17.
 */

public class OsmCameraPosition implements ICameraPosition{
    private final MapView osmMap;

    public OsmCameraPosition(MapView osmMap){
        this.osmMap = osmMap;
    }
    public float getZoom() {
        return osmMap.getZoomLevel();
    }

    public float getTilt() {
        return 90;
    }
}

File: app/src/main/java/nl/rsdt/japp/jotial/maps/wrapper/osm/OsmCameraPosition.kt
package nl.rsdt.japp.jotial.maps.wrapper.osm

import nl.rsdt.japp.jotial.maps.wrapper.ICameraPosition
import org.osmdroid.views.MapView

/**
 * Created by mattijn on 17/08/17.
 */

class OsmCameraPosition(private val osmMap: MapView) : ICameraPosition {
    override val zoom: Float
        get() = osmMap.zoomLevel.toFloat()

    override val tilt: Float
        get() = 90f
}
